js/leaflet-sidebar.js

```diff
@@ -116,11 +140,19 @@ L.Control.Sidebar = L.Control.extend({
         if (!L.DomUtil.hasClass(this._sidebar, 'collapsed')) {
             this.fire('closing');
             L.DomUtil.addClass(this._sidebar, 'collapsed');
+
+            if (this.options.closeButton) {
+                L.DomUtil.addClass(this.getCloseButton(), 'u-display-none');
+            }
         }
 
         return this;
     },
 
+    getCloseButton: function() {
+        return this._closeButton;
+    },
```

Comment:
any reason why the close button DOM node should be part of the public interface?